problem_builder/mcq.py

```diff
@@ -51,6 +51,16 @@ class MCQBlock(SubmittingXBlockMixin, QuestionnaireAbstractBlock):
     CATEGORY = 'pb-mcq'
     STUDIO_LABEL = _(u"Multiple Choice Question")
 
+    message = String(
+        display_name=_("Message"),
+        help=_(
+            "General feedback provided when submitting, it is not shown if "
+            "question selected by user its own feedback tip."
```

Comment:
"General feedback shown to the learner when submitting. "
"(This is not shown if there is a more specific feedback tip for the choice selected by the learner.)"

(The text you used above is not grammatically correct because a comma "," should not be used to separate two independent clauses)